### PR TITLE
[REVIEW] Programming courses in Nepali

### PR DESCRIPTION
**Pull request by SpandanNepal**
## What does this PR do?
There are several programming courses on the internet nowadays, but some of them are missing here. This PR adds good quality courses.

### Why is this valuable (or not)?
The sources have good feedback and views.

### How do we know it's really free?
All the contents are freely available on YouTube.

### For book lists, is it a book? For course lists, is it a course? etc.
No it is not a book, but links to different video sources.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ne.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 * [Java Programming - Tutorial - Nepali](https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT) - BigData IT
 * [JavaScript](https://www.youtube.com/playlist?list=PLckS_N3kOwFH-GCqCd6i-vPo-Z75DcOnc) - EverydayKarma 🇳🇵
-
+* [JavaScript Games](https://www.youtube.com/watch?v=OORUHkgg4IM&list=PLpwngcHZlPadAbdD_sFE_moH6RjgaTFCw) - CodingNepal
```
Comment (sohansai) [inline courses/free-courses-ne.md:17]:
```suggestion
* [JavaScript Games](https://www.youtube.com/playlist?list=PLpwngcHZlPadAbdD_sFE_moH6RjgaTFCw) - CodingNepal
```

**Comment by sohansai** [comment]
Check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems

**Comment by domalasherpa** [comment]
please add a new line before node js heading in line 19.

**Comment by SpandanNepal** [comment]
@domalasherpa thank you.

**Comment by eshellman** [comment]
There's no Nepali in these playlists. They seem to me to be more in the screencast genre - we have a list for those.

**Pull request closed without merging**